apps/opik-backend/src/main/java/com/comet/opik/domain/cost/SpanCostCalculator.java

```diff
@@ -0,0 +1,17 @@
+package com.comet.opik.domain.cost;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.Map;
+
+@UtilityClass
+class SpanCostCalculator {
+    public static double textGenerationCost(ModelPrice modelPrice, Map<String, Integer> usage) {
```

Comment:
If we need to guarantee `Decimal32(7)` this should be BigDecimal

Reply:
Updated